Packs/Core/Integrations/CortexCoreIR/README.md

```diff
@@ -3298,4 +3298,51 @@ Run a shell command on a specific endpoint and return its result.
 | echo | | dummy_id | 2.2.2.2 | name | STATUS_010_CONNECTED | COMPLETED_SUCCESSFULLY |
 | echo hello | hello | dummy_id | 2.2.2.2 | name | STATUS_010_CONNECTED | COMPLETED_SUCCESSFULLY |
 | echo |  | dummy_id2 | 11.11.11.11 | name2 | STATUS_010_CONNECTED | COMPLETED_SUCCESSFULLY |
-| echo hello | hello | dummy_id2 | 11.11.11.11 | name2 | STATUS_010_CONNECTED | COMPLETED_SUCCESSFULLY |
\ No newline at end of file
+| echo hello | hello | dummy_id2 | 11.11.11.11 | name2 | STATUS_010_CONNECTED | COMPLETED_SUCCESSFULLY |
+
+
+### core-add-indicator-rule
+
+***
+Upload IOCs Rule to XSIAM. 
+When `ioc_object` is defined, disregard any other provided arguments as `ioc_object` takes precedence.
```

Comment:
When the `ioc_object` is defined, disregard any other provided arguments, as `ioc_object` takes precedence.